builder/main.go

```diff
@@ -90,14 +56,27 @@ func main() {
 		"actuaciones": len(args.exp.Actuaciones),
 	}).Printf("finished")
 
+	blacklist := os.Getenv("BLACKLIST_REGEX")
```

Comment:
This mixes env vars and args to pass arguments.
We should choose one and stick towit.